jablib/src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -129,9 +129,7 @@ void findMissingLocalizationKeys() throws IOException {
         assertEquals(List.of(), missingKeys,
                 missingKeys.stream()
                            .map(key -> LocalizationKey.fromKey(key.getKey()))
-                           .map(key -> "%s=%s".formatted(
-                                   key.getEscapedPropertiesKey(),
-                                   key.getValueForEnglishPropertiesFile()))
+                           .map(key -> key.getEscapedPropertiesKey() + "=" + key.getValueForEnglishPropertiesFile())
```

Comment:
This is semantically equivalent

One has to modify `getEscapedPropertiesKey` instead.

Reply:
@jetbrains-junie 

This is semantically equivalent

One has to modify getEscapedPropertiesKey instead.